### PR TITLE
Hexaflake-change files permission

### PR DESCRIPTION
**Pull request by wangxiaoyuvvv**
已正确修改文件权限

### CODE REVIEW

File: inference/configs/llama2_7b_mmlu/configurations.yaml
Comment (shh2000) [inline inference/configs/llama2_7b_mmlu/configurations.yaml:1]: do not change this file. using llama2_7b_mmlu/vendor_configs/hexaflake_configurations.yaml as an alternative, like https://github.com/FlagOpen/FlagPerf/blob/main/inference/configs/llama2_7b_mmlu/vendor_config/nvidia_configurations.yaml

File: inference/configs/host.yaml
Comment (shh2000) [inline inference/configs/host.yaml:1]: do not change this file except the comments. you can add ACCE_CONTAINER_OPT like metax

File: inference/run_hexaflake.py
Comment (shh2000) [inline inference/run_hexaflake.py:1]: is there any difference between run_hecaflake.py and run_inference.py? what's the main purpose

File: inference/benchmarks/llama2_7b_mmlu/pytorch/__init__.py
Comment (shh2000) [inline inference/benchmarks/llama2_7b_mmlu/pytorch/__init__.py:1]: please send the log to my email, and add results at README. you can refer to https://github.com/FlagOpen/FlagPerf/pull/227/files

File: inference/run_hexaflake.py
Comment (wangxiaoyuvvv) [reply inference/run_hexaflake.py:1]: The file run_hexaflake.py is aligned with the run.py file, but the main difference is that it requires specifying a password when entering certain instructions.

**Pull request closed without merging**